libs/cart/state/src/reducers/cart-item-entities/cart-item-entities.reducer.ts

```diff
@@ -18,14 +18,20 @@ export function daffCartItemEntitiesReducer<
 	U extends DaffCartItemInput = DaffCartItemInput,
 	V extends DaffCart = DaffCart,
 >(
-  state = daffCartItemEntitiesAdapter<T>().getInitialState({ daffState: DaffCartItemStateEnum.Default }),
+  state = daffCartItemEntitiesAdapter<T>().getInitialState(),
```

Comment:
Why did you remove this arg? Did it not set a default value for `daffState`?

Reply:
Turns out it just did this:
```
{
    entities: {},
    ids: [],
    daffState: "default"
}
```
I replaced what this tried to do with setting `daffState: "default"` on each cart item on a `DaffCartLoadSuccess` and a `DaffCartItemListSuccess`